backend/core/src/main/resources/db/migration/V2020.09.01_15.58__alter_companies_table.sql

```diff
@@ -0,0 +1,2 @@
+alter table companies
```

Comment:
Change the name of the migration. 
Now it will break the migrations order [https://github.com/BinaryStudioAcademy/bsa-2020-feedgrabber/tree/dev/backend/core/src/main/resources/db/migration](url)